src/uhi/resources/histogram.schema.json

```
@@ -177,9 +177,16 @@
           "oneOf": [
             {
               "type": "string",
-              "description": "A path (URI?) to the integer bin data."
+              "description": "A path (URI?) to the floating point bin data."
```

```suggestion
              "description": "A path (URI?) to the integer bin data."
```

Copy and paste mistake